### ISSUE TITLE
Content Block Media Heading and Paragraph does not match visual design

### ISSUE THREAD

**Issue by kexuibm** [carbon-design-system/ibm-dotcom-library#1873]
v1.5.1-rc.1

In the Content Block Media pattern, the heading and paragraphs have the wrong font size when compared to latest Learn Page Visual Design.

https://ibm.box.com/s/td94d8jdb0odjbyohgbmuhvh8lkmem6b

**Comment by oliviaflory** [comment]
Hey @kexuibm, would you mind sharing a screen shot of what you're seeing and any specific values for the text you could share would be really helpful.

**Comment by photodow** [comment]
@kexuibm Can you provide more information, or a screenshot or something? from a quick glance it looks correct. @oliviaflory feel free to correct me.

http://ibmdotcom-react.mybluemix.net/?path=/story/patterns-blocks-contentblockmedia--default

**Comment by oliviaflory** [comment]
There is a known issue of the Carbon font package rendering the expressive headings in regular weight when they should be light weight, it does look like the Heading is rendering as regular which might be part of the issue @kexuibm is seeing. 

I believe they are working on fixing the weight issue @photodow . Does seem like the XL breakpoint is displaying at 28px when it should be 32px, but my inspect tool isn't cooperating fully.

**Comment by kexuibm** [comment]
Sure thing, here is what it looks like in the Visual Design and what it looks like in the DDS pattern:

![example pattern headline page](https://user-images.githubusercontent.com/55464272/77690225-53731b80-6f79-11ea-9123-ed5b4a36934c.png)
![example pattern headline design](https://user-images.githubusercontent.com/55464272/77690230-55d57580-6f79-11ea-9687-a760d2668204.png)

**Comment by oliviaflory** [comment]
Thanks @kexuibm that is really helpful

Looks like the Headline is rendering as `regular` weight, and in the design it is intended to be `light` weight. That is from the Carbon fonts package rendering incorrectly. We will double check if that has been fixed yet.

The paragraph is using `expressive-heading-03` which is 20px font on the DDS pattern, but we recently decided to update to `expressive-paragraph-01` like you've shown here in the Visual design. We should be updating the DDS patterns in our next sprint, we have a feature request open to update the patter: #1600
